Invoke-ScriptInRemoteSessions.ps1

```diff
@@ -0,0 +1,91 @@
+Param (
+    [Parameter(Mandatory = $false)] [String] $ScriptFileName = "Clear-ComputeNode.ps1",
+    [Parameter(Mandatory = $false)] [String] $Addresses = "127.0.0.1",
+    [Parameter(Mandatory = $false)] [Switch] $IndividualCredentials,
+    [Parameter(Mandatory = $false, ValueFromRemainingArguments = $true)] $ArgumentsToPass
+)
+
+function New-Sessions {
+    Param (
+        [Parameter(Mandatory=$true)] [String[]] $Addresses,
+        [Parameter(Mandatory=$false)] [Int] $RetryCount = 10,
+        [Parameter(Mandatory=$false)] [Int] $Timeout = 300000

```

Comment:
this is 5 minutes? way too long, we probably need to aim for like 5 seconds. Please include units (ms / seconds, whatever) in variable name.